src/curves.py

```diff
@@ -8,190 +8,190 @@
 
 class Curve(ABC):
 
-  lsys : ls.LSystem
-  turtle : mt.Turtle
-  startpoint : str
+    lsys: ls.LSystem
+    turtle: mt.Turtle
+    filename: str
+    postProcessMap: any
```

Comment:
very neat idea to collect such post-process-steps in one member-map! (small hint/TODO: snake-case for everything ... I have to add something to take care of that automatically -- suggestions?)

Reply:
Actually, one could do something completely different. All post process steps are actually similar rewriting steps as the main rewritting rules. I would propose to define everything as a pipeline, similar as in tensorflow for example:

```
pipeline = [
    dragoncurverwrite(iterations=4),
    postprocessrewrite(iterations=1),
    postprocessrewrite2(iterations=1)
]
```